shared/middleware/helpers/table.rb

```diff
@@ -32,9 +92,15 @@ def delete(id)
 
   def delete_all()
     items.delete
+    @metadata_dataset.delete if metadata_dataset
   end
```

Comment:
I think we don't want to delete the metadata when we delete_all. If the user imports & overwrites, we want to remember the column names.